src/index.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import T from 'prop-types';
-import { soundManager } from 'soundmanager2';
 
 const pendingCalls = [];
 let initialized = false;
 
+let soundManager;
+if (typeof window !== 'undefined') {
+  soundManager = require('soundmanager2');
```

Comment:
Sorry, one more that I spotted when testing this. You need to access the `.soundManager` property and assign it to the `soundManager` variable so that it works:

```js
soundManager = require('soundmanager2').soundManager;
```